backend/test/game/test_turn.py

```diff
@@ -0,0 +1,54 @@
+import collections
+import pytest
+
+import backend.src.game.turn as turn
+from backend.src.game.dice.dice import DieValue
+
+
+def test_less_than_three_victory_point_dice_no_victory_points_awarded():
+    dice = collections.Counter([DieValue.TWO, DieValue.TWO,
+                                DieValue.HEAL, DieValue.ATTACK,
+                                DieValue.HEAL, DieValue.ATTACK,
+                                ])
+    assert turn.calculate_victory_points_from_dice(dice) == 0
+
+
+def test_three_victory_point_dice_two_victory_points_awarded():
```

Comment:
I feel that the tests are missing the dice value that was rolled, ie
test_three_twos_rolled_two_victory_points_awarded()
By reading the test names alone, it doesn't make sense.